ql/src/java/org/apache/hadoop/hive/ql/DriverUtils.java

```diff
@@ -104,8 +110,7 @@ public static SessionState setUpSessionState(HiveConf conf, String user, boolean
         SessionState.start(sessionState);
       }
       SessionState.setCurrentSessionState(sessionState);
```

Comment:
Do you know if that is required? i think we can drop it since it's set inside `start`